Amplify/Categories/DataStore/Model/Internal/Model+Array.swift

```diff
@@ -7,6 +7,7 @@
 
 import Foundation
 
+/// Note that although this is public, it is intended for internal use and not consumed directly by host applications.
```

Comment:
Here and in all cases where you add a note to an extension wrapper: The warning needs to be on the public symbol, not the extension wrapper.